ui/packages/shared/components/src/MatchersInput/index.tsx

```diff
@@ -166,31 +263,85 @@ const MatchersInput = ({
   };
 
   const applySuggestion = (suggestionIndex: number): void => {
-    const suggestion = getSuggestion(suggestionIndex);
+    let suggestion = getSuggestion(suggestionIndex);
+
+    if (suggestion.type === 'labelValue') {
+      suggestion.value = addQuoteMarks(suggestion.value);
+    }
+
     const newValue = complete(suggestion);
     resetHighlight();
+
+    if (suggestion.type === 'labelName') {
```

Comment:
Shall we make these literals as constants? That will help us define even stricter type for this field.